src/components/BookCard.js

```diff
@@ -47,7 +47,7 @@ const BookCard = ({ book, index, onEdit, onComment }) => {
       </div>
       <div className="book-card-right">
         <p className="current-chapter">Current Chapter</p>
-        <p className="book-state">{book.state || 'Not Started'}</p>
+        <p className="book-state">{book.state || 'Chapter-1'}</p>
         <button type="button" onClick={handleUpdateProgress}>
           Update Progress
         </button>
```

Comment:
- Your project looks like the provided wireframe, :+1: :+1: But there are few changes you need to make before going to the next milestone. 

- [ ] Looking at your `.eslintrc.json`, it seems you are using the wrong or modified code, The content of the file is different from the one in the official Microverse Linters repository and it affects the Linters check, some errors in the code are not picked up by linters. I'll suggest you go to this repository [ [MICROVERSE LINTERS](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json) ] and copy the correct `.eslintrc.json` file content. Make sure to run the linters check after that and fix any error that your code has. NB: It is against Microverse rules to Modify any Linters code, the linters are there to make sure your code follows the best standard. :smile: :wave: :+1: 

## I'LL GREATLY SUGGEST YOU STOP MODIFING THE CONTENT OF ANY OF THE LINTERS FILES AS IT IS AGAINST THE MICROVERSE RULE.

Reply:
Thank you for your kind review and suggestions.
This issue is addressed and .eslintrc is restored back to the original copy as provided by Microverse.